.github/FUNDING.yml

```
@@ -0,0 +1,2 @@
+patreon: zipcaptions
+buy_me_a_coffee: zipcaptions
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)